packages/esm-patient-chart-app/src/encounter-tile/utils/helpers.ts

```diff
@@ -0,0 +1,146 @@
+import dayjs from 'dayjs';
+import { formatDate, parseDate } from '@openmrs/esm-framework';
+
+export function getEncounterValues(encounter, param: string, isDate?: Boolean) {
+  if (isDate) return dayjs(encounter[param]).format('DD-MMM-YYYY');
+  else return encounter[param] ? encounter[param] : '--';
+}
+
+export function formatDateTime(dateString: string): any {
+  const format = 'YYYY-MM-DDTHH:mm:ss';
+  if (dateString.includes('.')) {
+    dateString = dateString.split('.')[0];
+  }
+  return dayjs(dateString, format, true).toDate();
+}
+
+export function obsArrayDateComparator(left, right) {
+  return formatDateTime(right.obsDatetime) - formatDateTime(left.obsDatetime);
```

Comment:
Same here, we can leverage the framework functions, check on my PR, I got a way around this